cmake/configure_ace_tao.pl

```diff
@@ -100,6 +101,12 @@ sub read_file {
   my $platform_macros_path = "$values{ace}/include/makeinclude/platform_macros.GNU";
   open(my $platform_macros_file, '>', $platform_macros_path)
     or die("Failed to open $platform_macros_path: $!");
+  if ($values{compiler}) {
+    for my $var ('CC', 'CXX', 'LD') {
+      print $platform_macros_file ("$var = $values{compiler}\n");
+    }
+  }
+  print $platform_macros_file ("CCFLAGS += \$(opendds_cmake_std)\n");
```

Comment:
These look kind of odd with a parenthesized expression that's just a single string

Reply:
A bit, but it's how Perl requires parentheses to be used with `print`. It's not much different looking from `foreach` loops though.

Reply:
I guess was expecting no parens when they don't add meaning or clarity, but it's subjective.  Perl is TMTOWTDI to a fault.